.travis.yml

```diff
@@ -5,11 +5,18 @@ php:
   - 5.6
   - 7.0
 
-sudo: false
+dist: trusty
+sudo: required
```

Comment:
Neat. We can give the [new VM infrastructure](https://docs.travis-ci.com/user/trusty-ci-environment/) a try.